bundles/specmate-emfrest/src/com/specmate/emfrest/crud/CrudUtil.java

```diff
@@ -60,16 +62,53 @@
 		return new RestResult<>(Response.Status.OK, target, userName);
 	}
 
+	public static RestResult<?> recycle(Object target, String userName) {
+		EObject theTarget = (EObject) target;
+		SpecmateEcoreUtil.setAttributeValue(theTarget, true, "isRecycled");
+		SpecmateEcoreUtil.setAttributeValue(theTarget, true, "hasRecycledChildren");

```

Comment:
Coding the attribute names here could lead to issues when we decide to rename the attributes. Is there a way to use the a normal setter?